backend/server/src/utility/jwtHelper.ts

```diff
@@ -1,9 +1,10 @@
 import jwt from "jsonwebtoken";
 
 const SECRET_KEY: string = process.env.JWT_SECRET as string;
+const REFRESH_SECRET_KEY: string = process.env.JWT_REFRESH_SECRET as string;
 
-if (!SECRET_KEY) {
-  throw new Error("Missing JWT_SECRET environment variable");
+if (!SECRET_KEY || !REFRESH_SECRET_KEY) {
+  throw new Error('Missing JWT_SECRET or JWT_REFRESH_SECRET environment variables');
 }
 
 export function generateToken(payload: any, expiresIn: string = "1h"): string {
```

Comment:
I would use the types provided by [@types/jsonwebtoken](https://www.npmjs.com/package/@types/jsonwebtoken) instead of opting for `any`